### PR TITLE
Format Macrokiosk post body correctly as JSON

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -2100,19 +2100,19 @@ def send_macrokiosk_message(cls, channel, msg, text):
...
 
         url = 'https://www.etracker.cc/bulksms/send'
-        payload_json = json.dumps(payload)
+        payload = json.dumps(data)
 
         headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
         headers.update(TEMBA_HEADERS)
 
-        event = HttpEvent('POST', url, payload_json)
+        event = HttpEvent('POST', url, payload)
```
Comment (rowanseymour) [inline temba/channels/models.py]: Not seeing any actual difference here...

Comment (norkans7) [inline temba/channels/models.py]: renaming the variable is hiding the change in diff however https://github.com/nyaruka/rapidpro/pull/1278/files#diff-90054c9c29182e6b6d53ce8284d82e3dR2120
we were passing a dict and now we pass a dump of the dict

Comment (norkans7) [inline temba/channels/models.py]: Finally I have been able to send a SMS that they received at Macrokiosk, previously I was getting an error of failed parsing

Comment (rowanseymour) [inline temba/channels/models.py]: Doesn't requests.post dump the dictionary automatically?

Comment (norkans7) [inline temba/channels/models.py]: I thought so too but that was breaking

Comment (nicpottier) [inline temba/channels/models.py]: Passing the dictionary as `json` to `requests.post` will take care of the headers and encoding.

Comment (nicpottier) [inline temba/channels/models.py]: json named parameter that is.

Comment (norkans7) [inline temba/channels/models.py]: Ahh okay

**Pull request merged**